charts/cloudzero-prometheus-agent/templates/secret.yaml

```diff
@@ -11,5 +11,5 @@ metadata:
   name: {{ include "cloudzero-agent.secretName" .}}
   namespace: {{ include "cloudzero-agent.namespace" . }}
 data:
-  value: {{ .Values.credentials.api_key | b64enc | quote }}
+  value: {{ .Values.apiKey | b64enc | quote }}
```

Comment:
same here - are we putting this in the `cloudzero` block? maybe I just don't know what we decided on

Reply:
We removed the cloudzero block